addons/account/models/account_tax.py

```diff
@@ -795,27 +799,9 @@ def add_extra_base(other_batch, tax_data, sign):
                 # To compute the base amount of t2, we need to add back the tax amount of t1.
                 elif special_mode == 'total_excluded':
```

Comment:
Since we now handle all 3 `special_mode`, maybe it's better to write `else` with comment here just for more clarity. (same for the JS part too, but ignore it if you disagree)
```else:  # special_mode == 'total_excluded'```